configureWorkspace/configure_dotnetcore.ts

```diff
@@ -28,11 +28,19 @@ export const configureDotNetCoreConsole: IPlatformGeneratorInfo = {
     defaultPort: undefined
 };
 
-const AspNetCoreRuntimeImageFormat = "microsoft/aspnetcore:{0}.{1}{2}";
-const AspNetCoreSdkImageFormat = "microsoft/aspnetcore-build:{0}.{1}{2}";
-const DotNetCoreRuntimeImageFormat = "microsoft/dotnet:{0}.{1}-runtime{2}";
-const DotNetCoreAspNetRuntimeImageFormat = "microsoft/dotnet:{0}.{1}-aspnetcore-runtime{2}";
-const DotNetCoreSdkImageFormat = "microsoft/dotnet:{0}.{1}-sdk{2}";
+// .NET Core 1.0 - 2.0 images are published to Docker Hub Registry.
+const LegacyAspNetCoreRuntimeImageFormat = "microsoft/aspnetcore:{0}.{1}{2}";
+const LegacyAspNetCoreSdkImageFormat = "microsoft/aspnetcore-build:{0}.{1}{2}";
+const LegacyDotNetCoreRuntimeImageFormat = "microsoft/dotnet:{0}.{1}-runtime{2}";
+const LegacyDotNetCoreSdkImageFormat = "microsoft/dotnet:{0}.{1}-sdk{2}";
+
+// .NET Core 2.1+ images are now published to Microsoft Container Registry (MCR).
+// https://hub.docker.com/_/microsoft-dotnet-core-runtime/
+const DotNetCoreRuntimeImageFormat = "mcr.microsoft.com/dotnet/core/runtime:{0}.{1}{2}";
+// https://hub.docker.com/_/microsoft-dotnet-core-aspnet/
+const AspNetCoreRuntimeImageFormat = "mcr.microsoft.com/dotnet/core/aspnet:{0}.{1}{2}";
+// https://hub.docker.com/_/microsoft-dotnet-core-sdk/
+const DotNetCoreSdkImageFormat = "mcr.microsoft.com/dotnet/core/sdk:{0}.{1}{2}";
 
```

Comment:
What about `AspNetCoreSdkImageFormat`? Is the ImageSDKFormat the same for both Console and non-console?

Reply:
`AspNetCoreSdkImageFormat` is now deprecated. ASP.NET Core `1.0`, `1.1` and `2.0` used to require Node, Bower, and Gulp for building a Web App. Therefore, those tooling were included into the image `microsoft/aspnetcore-build` and weren't in `microsoft/dotnet:{major}.{minor}.{patch}-sdk`. 

Starting with 2.1, ASP.NET Core team has replaced the above tooling with a single one - [LibMan](https://devblogs.microsoft.com/aspnet/library-manager-client-side-content-manager-for-web-apps/). For apps that target .NET Core 2.1+ you need to use a single .NET Core SDK to build either Console App  or ASP.NET Core Web App.